README.md

```diff
@@ -1,5 +1,128 @@
+@@
+# WasmEdge
+
+WasmEdge is a lightweight, high-performance, and extensible WebAssembly runtime for cloud native, edge, and decentralized applications. It powers serverless apps, embedded functions, microservices, udf, smart contracts, and IoT devices. WasmEdge is currently a CNCF (Cloud Native Computing Foundation) Sandbox project.
+
 # WasmEdge Docs
 
-This repo contains technical documentation for the [WasmEdge Runtime](https://github.com/WasmEdge/WasmEdge) project. The documentation website built from this repo is published [here](https://wasmedge.org/docs/), check it out!
+This repository contains the technical documentation for the [WasmEdge Runtime](https://github.com/WasmEdge/WasmEdge) project. The documentation website built from this repo is published [here](https://wasmedge.org/docs/). It contains all the necessary information on what Was Edge Runtime is and how to Develop, Embed and Extend WasmEdge  for contributions.
```

Comment:
what Was Edge Runtime is --- grammar error

how to Develop, Embed and Extend WasmEdge  for contributions -- I'd like to say how to develop Wasm app, embed WasmEdge into your host applications, and extend/contribute to WasmEdge. It's more specific and can make readers more understand. Let me know what do you think.

Reply:
I have corrected the grammatical error. 
Yeah the later ones sounds more informative and is easy for the users to understand. 
I'll incorporate these changes and create another pull request.
Also, I wish to ask if there are any other details that might be necessary to be included in the readme.